prusti-viper/src/encoder/encoder.rs

```diff
@@ -789,14 +789,15 @@ impl<'v, 'tcx> Encoder<'v, 'tcx> {
         expr
     }
 
-    pub fn encode_item_name(&self, def_id: DefId) -> String {
-        let full_name = format!("m_{}", encode_identifier(self.env.get_item_def_path(def_id)));
-        let short_name = format!("m_{}", encode_identifier(
-            self.env.tcx().opt_item_name(def_id)
-                .map(|s| s.name.to_ident_string())
-                .unwrap_or_else(|| self.env.get_item_name(def_id))
-        ));
-        self.intern_viper_identifier(full_name, short_name)
+    pub fn encode_item_name(&self, def_id: DefId, prefix: &str) -> String {
```

Comment:
I think it would be better to have two methods `encode_item_name` that takes only `def_id` and `encode_item_name_with_prefix` that corresponds to this one.